ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -323,7 +323,8 @@ class TensorboardLogger(BaseLogger):
 
     Args:
         *args: Positional arguments accepted from :class:`~tensorboardx.SummaryWriter`.
-        **kwargs: Keyword arguments accepted from :class:`~tensorboardx.SummaryWriter`.
+        **kwargs: Keyword arguments accepted from :class:`~tensorboardx.SummaryWriter`, for example,
+        `log_dir` to setup path to the directory where to log.
```

Comment:
You need to indent this line 4 spaces in order to continue as the previous in the rendered doc